### ISSUE TITLE
User - Preferences - Timeline visuals

### ISSUE THREAD

**Issue by mjy** [SpeciesFileGroup/taxonworks#1332]
* [ ] Origin
* [ ]

**Comment by adrik29** [comment]
As I am certain there are different opinions and preferences among the users, may I suggest that those preferences are customizable/toggled?

**Comment by mjy** [comment]
Yes, that's the idea.  Hopefully though we can come up with small number, rather than a massive number.  For example, what might the 5 most important settings be.

You'll hear me say this a lot- each setting should help to answer a question.  Spelling out the question, e.g. as "*nomenclatural curator* i need *origin displayed* so I can *determine the origin of the citation in relation to the data it is attached to and I can fix the citation*" will be very helpful.

**Comment by proceps** [comment]
If talking about top 5, I would vote to toggle all quick-recent-pinboard-search interfaces to a search mode. I could find only very few occasions when recent o quick helped me. In most cases they only take display space and require extra mouse clicks for me to enter information. Edit nomenclature probably the only place where quick is useful for relationships or statuses, since those have predefined values.

**Comment by mjy** [comment]
@proceps sorry, I didn't properly describe this (your stuff is good- maybe a different issues).

This is specifically for the options here:

![Screen Shot 2020-04-10 at 12 08 02 PM](https://user-images.githubusercontent.com/22600/79008925-028c2700-7b24-11ea-977d-9eb6ef5dfcc1.png)

or here (the two may ultimately merge to the same thing)

![Screen Shot 2020-04-10 at 12 08 25 PM](https://user-images.githubusercontent.com/22600/79008929-0455ea80-7b24-11ea-9196-29cda18ad73d.png)

**Comment by adrik29** [comment]
In my case, as I'm preparing a nomenclatural catalogue, it is fantastic to have the logonymy structured so I can immediately know what's going on:
![image](https://user-images.githubusercontent.com/46284943/79027411-79e0ab80-7b62-11ea-945d-d9d2fb6a86d7.png)

This structure mimics fairly well the structured logonymy that I use in the text of my catalogue:
![image](https://user-images.githubusercontent.com/46284943/79027459-aa284a00-7b62-11ea-95b4-4cece98395db.png)
